### PR TITLE
Addressing https://github.com/Polymer/polymer/issues/4463

### PR DESCRIPTION
**Pull request by katejeffreys**

### CODE REVIEW

File: app/2.0/docs/devguide/style-shadow-dom.md [hunk 1]
```diff
@@ -387,9 +387,12 @@ You can select by class:
 ```html
 <dom-module id="x-foo">
   <template>
-    <style>
-      ::slotted(.green) {
-        color: green;
+    <style> 
+      <!-- To work within the Shady CSS scoping shim limitations,
+           and to ensure consistent cross-browser behavior,
+	   add a selector to the left of the ::slotted(.classname) notation. -->
```
Comment (arthurevans) [inline app/2.0/docs/devguide/style-shadow-dom.md]: Note that this applies to _any_ `::slotted()` selector, so it should be moved up to the main discussion of `::slotted()`, and shown in all of the examples.

Comment (katejeffreys) [inline app/2.0/docs/devguide/style-shadow-dom.md]: Can you have a look at this one: http://plnkr.co/edit/suVhRDAB5Rq7Q7T6zbMI?p=preview

Comment (katejeffreys) [inline app/2.0/docs/devguide/style-shadow-dom.md]: Also this one, in which adding a selector to the left of the ::slotted syntax seems to break the styling completely http://plnkr.co/edit/ioTo3XjgrTwAzcCHS1zc?p=preview

**Comment by katejeffreys** [comment]
Thanks @arthurevans

File: app/2.0/docs/devguide/style-shadow-dom.md [hunk 2]
```diff
@@ -274,54 +274,49 @@ Styling with the `:host` selector is one of two instances where rules inside a s
...
-[See it on Plunker](http://plnkr.co/edit/cGrbntuantGtqj7Poqft?p=preview)
+[See it on Plunker](http://plnkr.co/edit/uA4bHmAFWgFjOSghEFeT?p=preview)
 
 `x-foo.html` { .caption}
 ```html
 <dom-module id="x-foo">
   <template>
     <style>
-      ::slotted(*) {
+      p, h1 ::slotted(*) {
```
Comment (arthurevans) [inline app/2.0/docs/devguide/style-shadow-dom.md]: Sorry I missed this before: This rule may not be doing what you expect it to do. 
The comma is an "or" operator, but it operates on an entire selector. In other words: `p, h1 ::slotted(*) { ... } ` is equivalent to:

```
p { ... } 
h1 ::slotted(*) { ...}
```

And not as you might think:

```
p ::slotted(*) { ... } 
h1 ::slotted(*) { ...}
```

Comment (arthurevans) [inline app/2.0/docs/devguide/style-shadow-dom.md]: If you want the latter, you can use:

```
p ::slotted(*), h1 ::slotted(*) { ... }
```

**Comment by katejeffreys** [comment]
http://plnkr.co/edit/pb0D6r15jvvxYVWsZ95U?p=preview

File: app/2.0/docs/devguide/style-shadow-dom.md [hunk 3]
```diff
@@ -274,54 +274,49 @@ Styling with the `:host` selector is one of two instances where rules inside a s
 
 ### Style slotted content (distributed children)
 
-You can create slots in an element's template that are populated at runtime using this syntax:
+You can create **slots** in an element's template that are populated at runtime. For more information on slots, see the documentation on [/2.0/docs/devguide/shadow-dom#shadow-dom-and-composition](shadow DOM and composition).
```
Comment (arthurevans) [inline app/2.0/docs/devguide/style-shadow-dom.md]: Ack sorry one more fix—this link has the text & url reversed.

**Pull request merged**